blog/dec-quiet-month-22.md

```diff
@@ -0,0 +1,41 @@
+---
+title: A Quiet Place (Dec'22)
+date: 2022-11-18T00:00:00.000Z
+authors:
+  name: plainekevin
+  url: 'https://github.com/PlaineKevin'
+  image_url: 'https://github.com/plainekevin.png?size=96'
+slug: a-quiet-place-22
+---
+
+The Electron project will pause for the month of December 2022, then return to full speed in January 2023.
+
+<div style={{width:"100%",height:"0",paddingBottom:"42%",position:"relative"}}>
+  <iframe src="https://giphy.com/embed/lbQQ1yeN4Z2wGm9q4d" width="480" height="270" frameBorder="0" class="giphy-embed" allowFullScreen></iframe>
+</div>
+<p><a href="https://giphy.com/gifs/marvelstudios-marvel-loki-studios-lbQQ1yeN4Z2wGm9q4d">via GIPHY</a></p>
+
+---
+
+## What will be the same in December
+
+1. Zero-day and other major security-related releases will be published as necessary. Security incidents should be reported via [SECURITY.md](https://github.com/electron/electron/tree/master/SECURITY.md).
+1. [Code of Conduct](https://github.com/electron/electron/blob/main/CODE_OF_CONDUCT.md) reports and moderation will continue.
+
+## What will be different in December
+
+1. No new Stable releases in December. No Nightly and Alpha releases for the last two weeks of December.
+1. With few exceptions, no pull request reviews or merges.
+1. No issue tracker updates on any repositories.
+1. No Discord debugging help from maintainers.
+1. No social media content updates.
+
+## Why is this happening?
+
+With the success of December Quiet Month 2021, we wanted to bring it back for 2022. December continues to be a quiet month for most companies, so we want to give our maintainers a chance to recharge. We encourage other projects to consider similar measures.
+
+## Should I be worried about the future of Electron?
```

Comment:
Is this necessary this year since we had the same thing happen last year without any concerns?

Reply:
That's fair. Let me move some of this to the section above and remove this section as it's probably not necessarily